shared/reducers/chat.js

```diff
@@ -348,6 +348,18 @@ function reducer(state: Constants.State = initialState, action: Constants.Action
         )
       )
     }
+    case 'chat:setTypers': {
+      const {conversationIDKey, typing} = action.payload
+      // $FlowIssue
+      return state.update('conversationStates', conversationStates =>
+        updateConversation(
+          conversationStates,
+          conversationIDKey,
+          // $FlowIssue
+          conversation => conversation.update('typing', _ => Set(typing))
```

Comment:
Oh!  Sure thing.  Pushed.  Also going to stick with Set() in the reducer, had been going back and forth on which to use.